src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -86,8 +89,8 @@
           ^{:key (str network)}
           [quo/text
```

Comment:
not for this pr, but we should probably extract this to a helper of some form, it's used all over the wallet.